tests/fsharp/Compiler/CodeGen/EmittedIL/CeEdiThrow.fs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+namespace FSharp.Compiler.UnitTests.CodeGen.EmittedIL
+
+open FSharp.Compiler.UnitTests
+open NUnit.Framework
+
+#if NETCOREAPP
```

Comment:
Is the reason for guarding this test with the ifdef related to the IL that gets emitted being different on .NET Framework?

Reply:
Right, no not directly I just thought it was an easy way to exclude netfx versions that don't ship the type, which would cause throw to be emitted instead. But I should have asked, which versions do you run these tests on :)

Reply:
All build and test is don against `net472`, since that's the base framework version that VS depends on. So this should be fine.

Reply:
I'll remove the ifdef!

Reply:
So apparently EDI.Throw(exn) doesn't exist on netfx after all. I thought I had checked fairly carefully, so I did it again and the link does say the docs apply to netfx too, so I'm not sure what's broken there.

https://docs.microsoft.com/en-us/dotnet/api/system.runtime.exceptionservices.exceptiondispatchinfo.throw?view=netcore-3.1#System_Runtime_ExceptionServices_ExceptionDispatchInfo_Throw_System_Exception_

Anyway the workaround is to move to `EDI.Capture(exn).Throw()` instead, which I have done.